### PR TITLE
Add comment and edit the transpose commend

### PR DESCRIPTION
**Pull request by 4D42**
Small pull request to add a small comment to try to explain why the reshaping and transposing and also edit the transpose parameter to make a bit more sense

### CODE REVIEW

File: CaImAn/caiman_main.py [hunk 1]
```diff
@@ -198,10 +198,11 @@ def save_caiman_to_doric(
     shape = np.array(Y).shape
     shape = (shape[1],shape[2], shape[0])
 
-    AC = (A.dot(C)).reshape(shape, order='F').transpose((-1, 0, 1))
-    Y  = Y.reshape(shape, order='F').transpose((-1, 0, 1))
+    # Because of the Fortran ordering it need to be reshaped in a specific shape(..., ..., Time) then transposed to (Time, ..., ...) to be saved
+    AC = (A.dot(C)).reshape(shape, order = 'F').transpose((2, 0, 1))
+    Y  = Y.reshape(shape, order = 'F').transpose((2, 0, 1))
     A = A.toarray()
```
Comment (doriclenses) [inline CaImAn/caiman_main.py:204]: do we need `.toarray()`?

File: CaImAn/caiman_main.py [hunk 2]
```diff
@@ -198,10 +198,11 @@ def save_caiman_to_doric(
     shape = np.array(Y).shape
     shape = (shape[1],shape[2], shape[0])
 
-    AC = (A.dot(C)).reshape(shape, order='F').transpose((-1, 0, 1))
-    Y  = Y.reshape(shape, order='F').transpose((-1, 0, 1))
+    # Because of the Fortran ordering it need to be reshaped in a specific shape(..., ..., Time) then transposed to (Time, ..., ...) to be saved
```
Comment (doriclenses) [inline CaImAn/caiman_main.py:201]: `# Before saving, the arrays have to be reshaped with Fortran order and shape (..., ..., Time), then transposed to (Time, ..., ...)`

Comment (doriclenses) [reply CaImAn/caiman_main.py:201]: what are (..., ..., Time) and (Time, ..., ...)? (Width, Height, Time) or (Height, Width, Time), ...?

**Review by doriclenses — CHANGES_REQUESTED** [review]
resolve comments

File: CaImAn/caiman_main.py (hunk 1 above)
Comment (4D42) [reply CaImAn/caiman_main.py:204]: apparently yeap :(

Comment (doriclenses) [reply CaImAn/caiman_main.py:204]: oki

**Review by doriclenses — APPROVED** [review]

**Review by kapilseh — APPROVED** [review]

**Pull request merged**